### PR TITLE
Added sample on how to create nested content programmatically

### PR DESCRIPTION
**Pull request by Jay-umbr**
To help out with https://github.com/umbraco/UmbracoDocs/issues/2714 , I added a sample on how to create nested content using ContentService in v8

### CODE REVIEW

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md [hunk 1]
```diff
@@ -136,3 +136,36 @@ Example:
 }
 <h3>@item.Value("heading")</h3>
 ```
+
+## Creating Nested Content programmatically
+
+For the sake of this example, let us assume we have a Nested Content property with alias `attendeeList`, where the element Document Type has an alias of `attendee`.
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:142]:
```suggestion
For the sake of this example, let us assume we have a Nested Content property with alias `attendeeList`, where the Element Document Type has an alias of `attendee`.
```

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md [hunk 2]
```diff
@@ -136,3 +136,36 @@ Example:
...
+It has the Properties: `user_name`, `user_email`, `join_time`, `leave_time`, `duration`, `phone`.
+
+To save data in Nested Content, we need to create a new C# List containing a `Dictionary` of type `<string, object>`. `Dictionary<string, string>` would also work.
+The first dictionary item property/parameter we should specify for each Nested Content element is `ncContentTypeAlias`, which is the alias of the Document Type that is the embed element in Nested Content.
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:146]:
```suggestion

The first dictionary item property/parameter we should specify for each Nested Content element is `ncContentTypeAlias`, which is the alias of the Document Type that is the embedded element in Nested Content.
```

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md [hunk 3]
```diff
@@ -136,3 +136,36 @@ Example:
...
+
+Afterwards, the entire list needs to be serialized to Json via JsonConvert.
+
+After injecting [ContentService](../../../../../Reference/Management/Services/ContentService/) and creating a new Content Node, we can do the following:
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:150]: Is this line necessary still, after you've added a variable for ContentService in the code sample?

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md [hunk 4]
```diff
@@ -136,3 +136,36 @@ Example:
...
+
+```csharp
+ var contentService = Services.ContentService; //if the class containing our code inherits SurfaceController, UmbracoApiController, or UmbracoAuthorizedApiController, we can get ContentService from Services namespace
+ IContent request = ContentService.Create("new node", guid, "mydoctype", -1); //here we create a new node, and fill out attendeeList afterwards
+ var attendees = new List<Dictionary<string, string>>(); //our list which will contain nested content
+ foreach (var person in participants) //participants is our list of attendees - multiple items, good use case for nested content
+            attendees.Add(new Dictionary<string, string>() {
+            {"ncContentTypeAlias","attendee"}, //this is the only "default" value we need to fill for nested item
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:158]: Sounds like this is something that is required when working with Nested Content - is that correct?
If that's indeed the case, perhaps this `ncContentTypeAlias` should be explained a bit more? 🤔

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md [hunk 5]
```diff
@@ -136,3 +136,36 @@ Example:
...
+            {"leave_time",person.leave_time.ToString()},
+            {"duration",person.duration.ToString()},
+            {"phone",person.phone.ToString()}
+            });
+            }
+            request.SetValue("attendeeList", JsonConvert.SerializeObject(attendees)); //bind the attendees List to attendeeList property on the newly created content node
+            ContentService.SaveAndPublish(request); //Save the entire node via ContentService
+```
+
+Where we iterate through a list of participants (the data could be coming from an API, for example) and add a new `Dictionary` item for each person in the list.
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:171]: The structure of this sentence is a bit strange.
Is it supposed to be a "second part" of another sentence before the code snippet perhaps? 🤔

**Review by sofietoft — CHANGES_REQUESTED** [review]
Looks great @Jay-umbr !

I've added a couple of comments and suggestions.
Take a look, and tell me what you think.

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md (hunk 3 above)
Comment (Jay-umbr) [reply Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:151]: I suppose not - will delete it later

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md (hunk 5 above)
Comment (Jay-umbr) [reply Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md]: Will look into it!

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md (hunk 4 above)
Comment (Jay-umbr) [reply Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:159]: ncContentTypeAlias is just the alias of the document type - can I explain it this way

Comment (sofietoft) [reply Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:159]: Ah. But it's the name of the alias when it is a Nested Content Element type - right?

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md (hunk 3 above)
Comment (Jay-umbr) [reply Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Nested-Content/index.md:151]: Removed that line in the latest commit 👍

**Review by sofietoft — APPROVED** [review]
Woohoo! Looks great @Jay-umbr 😄 🎉 

I'll merge it right in 💪

**Pull request merged**